raster/r.patch/main.c

```diff
@@ -180,7 +180,17 @@ int main(int argc, char *argv[])
     nrows = Rast_window_rows();
     ncols = Rast_window_cols();
 
-    bufrows = atoi(memory->answer) * (((1 << 20) / out_cell_size) / ncols);
+    /* memory reserved for presult and patch */
+    in_buf_size = out_cell_size * ncols * nprocs * 2;
```

Comment:
"reserved for result? and patch" or just "reserved for patch". I don't get the result part.

Reply:
`presult` is the preliminary result (initially the first raster map), see https://github.com/metzm/grass/blob/openmp_buffered_rows/raster/r.patch/main.c#L233
`patch` are the remaining input maps, see https://github.com/metzm/grass/blob/openmp_buffered_rows/raster/r.patch/main.c#L250
`presult` and `patch` are allocated here: https://github.com/metzm/grass/blob/e0b4aa9db08a8a067fd633273ee2098bedf1739a/raster/r.patch/main.c#L170-L177